### ISSUE TITLE
Missing braces

### ISSUE THREAD

**Issue by ethers** [ConsenSys/MultiSigWallet#11]
```
        for (uint i=0; i<owners.length; i++)
            if (confirmations[transactionHash][owners[i]])
                count += 1;
            if (count == required)
                return true;
```
Only the first "if" is inside the loop, but the 2nd "if" is poorly whitespaced and aligned with the first "if".
Braces usually improve readability and please use them, especially when nesting is involved, to improve clarity.

Another example, a brace for the for loop would also give more spacing from this monolithic block of code, and is probably more readable.
```
        for (uint i=0; i<transactionList.length; i++)
            if (   isPending && !transactions[transactionList[i]].executed
                || !isPending && transactions[transactionList[i]].executed)
            {
                _transactionListTemp[count] = transactionList[i];
                count += 1;
            }
        _transactionList = new bytes32[](count);
        for (i=0; i<count; i++)
            if (_transactionListTemp[i] > 0)
                _transactionList[i] = _transactionListTemp[i];
```

**Comment by Georgi87** [comment]
Agree on the first one, braces were missing. It was intended to have both if statements inside the for block.

Don't really agree on the second one. I think it is still readable and code is ok. But that's subjective.

**Comment by Georgi87** [comment]
Thanks for finding this issue. fixed in  b43b8d6a535c56e9d6ac720ac3f5448eb84f7431
